Pagination events

idanpeduiem opened the pull request.

idanpeduiem committed:
add .env to project

idanpeduiem committed:
change CreateEvent.ts name to event.ts

idanpeduiem committed:
some changes to make types work well

idanpeduiem committed:
add types

idanpeduiem committed:
Merge branch 'dev' into pagination-events

idanpeduiem committed:
finish pagination

idanpeduiem committed:
some types fixing and more

thcr1 commented on client/src/components/Events.tsx:
```diff
@@ -1,32 +1,88 @@
+          </Grid>
+        ))}
+      </GridHiddenScroll>
+      {/*{fetching && (*/}
```
remove commented code

idanpeduiem replied:
removed

thcr1 commented on server/src/server.ts:
```diff
@@ -16,22 +16,6 @@ import setUpMongo from "../mongo/mongoDbManager";
 
-  app.use(cookieParser());
-  
-  app.use(cors({
```
you are right that it shouldn't be like that, but we still need cors

idanpeduiem replied:
yoga add these cors by default, i dont think we need more cors then that but maybe i miss something so let me know if more cors needed
![image](https://user-images.githubusercontent.com/62710926/235793217-b19e763a-f012-4d22-8836-ceb9d9c94ae0.png)

docs:
https://the-guild.dev/graphql/yoga-server/docs/features/cors

idanpeduiem committed:
Merge remote-tracking branch 'origin/dev' into pagination-events

# Conflicts:
#	client/package-lock.json
#	client/src/components/CreateEvent.tsx
#	server/package-lock.json
#	server/src/graphql/resolvers/event/CreateEvent.ts

idanpeduiem committed:
add package-lock
this need to be removed - there is peer deps between react-qr-reader to react^18
